### PR TITLE
Language uk, pl and ru

### PR DESCRIPTION
**Pull request by mkikets99**
Added Polish, Russian and Ukrainian translation

### CODE REVIEW

File: lang/pl/zoom.php [hunk 1]
```diff
@@ -0,0 +1,169 @@
...
+$string['apisecret_desc'] = '';
+$string['apiurl'] = 'Link do Zoom API';
+$string['apiurl_desc'] = '';
+$string['audio_both'] = 'VoIP i Telephonia';
+$string['audio_telephony'] = 'Tylko telephonia';
+$string['audio_voip'] = 'Tylko VoIP';
+$string['cachedef_zoomid'] = 'ID usera Zoom';
+$string['cachedef_sessions'] = 'Informacja-raport z zoom o userach';
+$string['calendardescriptionURL'] = 'Link dla połączenia: {$a}.';
+$string['calendardescriptionintro'] = "\Opis:\n{\$a}";
```
Comment (jrchamp) [inline lang/pl/zoom.php:45]: Hi @mkikets99 

Should this be?
```suggestion
$string['calendardescriptionintro'] = "\nOpis:\n{\$a}";
```

**Comment by rlorenzo** [comment]
@mkikets99 Thanks! I merged to our master and the changes will be in our next release.

File: lang/pl/zoom.php (hunk 1 above)
Comment (mkikets99) [reply lang/pl/zoom.php]: Yes, sorry...
Missed that...

**Comment by rlorenzo** [comment]
@mkikets99 I just found out that the better way to contribute translations is to use https://lang.moodle.org

See that this plugin is already translated into many other languages: https://moodle.org/plugins/translations.php?plugin=mod_zoom

If you want to contribute future translations or updates, please do the following:

1. Create account at https://lang.moodle.org and login
2. Click on AMOS translator
3. Choose language(s) you want to translate
4. In Components section enter in the "filter" search "zoom"
5. Click "Save filter settings"
6. Make your changes and then click "Go to the stage" and you can submit your changes that way

**Pull request merged**